Dockerfile

```diff
@@ -15,11 +15,14 @@ RUN cd / && \
     chmod +x jq
 
 RUN mkdir -p /go/src/github.com/tikv/pd
+RUN mkdir -p /go/src/github.com/tikv/pd/client
 WORKDIR /go/src/github.com/tikv/pd
 
 # Cache dependencies
 COPY go.mod .
 COPY go.sum .
+COPY client/go.mod ./client
+COPY client/go.sum ./client
```

Comment:
why?

Reply:
Due to https://github.com/tikv/pd/actions/runs/5505647235/jobs/10033286083